app/controllers/obituaries/recent_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Obituaries::RecentController < ApplicationController
   def index
+    @obituaries = Obituary.all.order(:created_at)
```

Comment:
This is sort of a placeholder, and will likely need to be updated based on NYT data, which has publication_year instead of created_at. That is, if we want to intermingle user content and NYT, which I guess is TBD though I lean toward yes.

Reply:
Nice. Yeah it may be difficult to intermingle the user content and NYT response and still order by date cause they may be in different formats.. but I'm definitely down to give it a try.

Reply:
great point, Jack!